src/v2/view-builder/views/email/ChallengeAuthenticatorEmailView.js

```diff
@@ -1,107 +1,49 @@
-import { loc, View, createCallout, _ } from 'okta';
-import BaseForm from '../../internals/BaseForm';
-import BaseView from '../../internals/BaseView';
-import email from '../shared/email';
-import polling from '../shared/polling';
-import BaseAuthenticatorView from '../../components/BaseAuthenticatorView';
+import { View, _ } from 'okta';
+import hbs from 'handlebars-inline-precompile';
 import AuthenticatorVerifyFooter from '../../components/AuthenticatorVerifyFooter';
-import { SHOW_RESEND_TIMEOUT } from '../../utils/Constants';
+import BaseAuthenticatorEmailView from './BaseAuthenticatorEmailView';
 
-const ResendView = View.extend(
-  {
-    className: 'hide resend-email-view',
-    events: {
-      'click a.resend-link' : 'handelResendLink'
-    },
+const BaseAuthenticatorEmailForm = BaseAuthenticatorEmailView.prototype.Body;
 
-    initialize () {
-      this.add(createCallout({
-        content: `${loc('email.code.not.received', 'login')}
-        <a class='resend-link'>${loc('email.button.resend', 'login')}</a>`,
-        type: 'warning',
-      }));
-    },
-
-    handelResendLink () {
-      this.options.appState.trigger('invokeAction', 'currentAuthenticatorEnrollment-resend');
-      // Hide warning, but reinitiate to show warning again after some threshold of polling
-      if (!this.el.classList.contains('hide')) {
-        this.el.classList.add('hide');
-      }
-      this.showCalloutWithDelay();
-    },
-
-    postRender () {
-      this.showCalloutWithDelay();
-    },
-
-    showCalloutWithDelay () {
-      this.showMeTimeout = _.delay(() => {
-        this.$el.removeClass('hide');
-      }, SHOW_RESEND_TIMEOUT);
-    },
-
-    remove () {
-      View.prototype.remove.apply(this, arguments);
-      clearTimeout(this.showMeTimeout);
-    }
+// Courage doesn't support HTML, hence creating a subtitle here.
+const CheckYourEmailTitle = View.extend({
+  className: 'okta-form-subtitle',
+  attributes: {
+    'data-se': 'o-form-explain',
   },
-);
-
-const Body = BaseForm.extend(Object.assign(
-  {
-    save () {
-      return loc('mfa.challenge.verify', 'login');
-    },
-    initialize () {
-      BaseForm.prototype.initialize.apply(this, arguments);
-      const sendText = loc('oie.email.verify.sentText', 'login');
-      const enterCodeText = loc('email.mfa.email.sent.description.emailCodeText', 'login');
+  template: hbs`
+    {{i18n code="oie.email.verify.sentText" bundle="login"}}&nbsp;
+    <span class="strong">{{email}}.</span>
+    &nbsp;{{i18n code="email.mfa.email.sent.description.emailCodeText" bundle="login"}}
+  `,
+
+  getTemplateData () {
+    return _.pick(this.options, 'email');
```

Comment:
importing `_` just for this seems like an overkill. I think it would be better to just do ` return this.options && this.options.email;` and remove the `_` import.